docs/samples/fluid/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM pytorch/pytorch:2.0.0-cuda11.7-cudnn8-runtime
+
+COPY requirements.txt ./requirements.txt
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+COPY models.py /app/models.py
+ENV PYTHONPATH=/app/
+WORKDIR /app
+
+ENV MODEL_URL="/mnt/models" \
+    LOAD_IN_8BIT="False" \
+    GPU_ENABLED="False" \
+    MODEL_NAME="bloom"
+
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/cuda-11.7/targets/x86_64-linux/lib"
+
+CMD python models.py --model_name $MODEL_NAME
```

Comment:
How about adding a script that splits the whole process into two steps:

1. Copy the models folder if it is empty.
2. Start the model service.

Like below:

```bash
#!/usr/bin/env bash

# Define a timestamp function
timestamp() {
  date +"%T" # current time
}

echo Begin loading models at $(timestamp)

# Check if /app/models directory is empty
if [ ! "$(ls -A /app/models)" ]; then
  time cp -r /mnt/models /app/models
fi

python models.py --model_name $MODEL_NAME

echo Finish loading models at $(timestamp)
```

The purpose is to compare the model download and access time?

Reply:
Hi @cheyang , I test in this way in the beginning, however, it cannot show the actual performance improvement, as `cp` is not necessary here.

For KServe + Storage initiazlier: 
1. storage initializer downloads model to `/mnt/models`
2. application loads model from `/mnt/models`

For KServe + Fluid:
1. application loads model from `/mnt/models` (model is mounted by PVC, storage initializer is not needed any more)

The Fluid helps to remove the storage initializer step(two steps -> one step) and loading the model from the cache cluster is faster than loading from the default `/mnt/models` (EmptyDir) when it comes to large model.

Reply:
Got it!